package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/MeshImporter.cs

```diff
@@ -337,64 +356,78 @@ public class GeometrySubsets {
           // TODO: this should be moved to C++
           int[] offsetMapping = new int[usdMesh.faceVertexCounts.Length];
           int curOffset = 0;
-          for (int i = 0; i < usdMesh.faceVertexCounts.Length; i++) {
+          for (int i = 0; i < usdMesh.faceVertexCounts.Length; i++)
+          {
             offsetMapping[i] = curOffset;
             curOffset += Math.Max(0, usdMesh.faceVertexCounts[i] - 3);
           }
 
           var newSubsets = new GeometrySubsets();
-          foreach (var nameAndSubset in geomSubsets.Subsets) {
+          foreach (var nameAndSubset in geomSubsets.Subsets)
+          {
             var newFaceIndices = new List<int>();
-            foreach (var faceIndex in nameAndSubset.Value) {
+            foreach (var faceIndex in nameAndSubset.Value)
+            {
               newFaceIndices.Add(faceIndex + offsetMapping[faceIndex]);
-              for (int i = 1; i < usdMesh.faceVertexCounts[faceIndex] - 2; i++) {
+              for (int i = 1; i < usdMesh.faceVertexCounts[faceIndex] - 2; i++)
+              {
```

Comment:
You should definitely test an asset that has geometry subsets.

Reply:
Do we have a known one?